createst.py

```diff
@@ -141,10 +141,20 @@ def write_to_file(data):
         sys.exit(1)
     with open(test_yaml_path, "w+") as fp:
         fp.write("# *** Add configuration here ***\n\n")
-        if not args["strictcsums"]:
-            fp.write("args:\n- -k none\n\n")
+        if check_set_args():
+            fp.write("args:\n")
+            if not args["strictcsums"]:
+                fp.write("- -k none\n")
+            if args["midstream"]:
+                fp.write("- --set stream.midstream=true\n")
+            fp.write("\n")
         fp.write(data)
 
+def check_set_args():
+    features = ["midstream"]
```

Comment:
This should probably also have `strictcsums`.

Reply:
I first included it, but then I realized we maybe don't want to have it here, because when the user passes `strictcsums`, `-k none` won't be used as an argument, so my understanding was that we will have that argument when strictcsums is false, only.